README.md

```diff
@@ -1,5 +1,114 @@
 
 # OSDP Prototype
 
- - [CDK Stack](osdp/README.md)
- - [IIIF manifest fetcher docker image](iiif/README.md)
\ No newline at end of file
+## IIIF manifest fetcher docker image
+
+ - [IIIF manifest fetcher docker image](iiif/README.md)
+
+## OSDP Prototype Stack (CDK)
+
+All steps performed from `osdp` directory
+
+```
+cd osdp
+```
+
+### Setup
+
+To manually create a virtualenv
+
+```
+$ python3 -m venv .venv
+```
+
+Aactivate your virtualenv.
+
+```
+$ source .venv/bin/activate
+```
+
+Install the required dependencies.
+
+```
+$ pip install -r requirements.txt requirements-dev.txt
```

Comment:
This should be

```
$ pip install -r requirements.txt -r requirements-dev.txt
```

With the extra `-r` flag.

Reply:
Done. Thanks.